tests/fixture-valid.js

```diff
@@ -369,17 +393,17 @@ function checkPanTiltMaxInPhysical(channel, mode) {
 }
 
 function checkUnusedChannels() {
-  const unused = definedChannelKeys.filter(
-    chKey => !usedChannelKeys.includes(chKey)
+  const unused = [...definedChannelKeys].filter(
+    chKey => !usedChannelKeys.has(chKey)
   );
 
   if (unused.length > 0) {
-    result.warnings.push(`Channels ${unused} defined but never used.`);
+    result.warnings.push('Unused channel(s): ' + unused.join(', '));
   }
 }
 
 // returns whether the object contains data
-function checkIsEmpty(obj, messageIfEmpty) {
+function assumeNotEmpty(obj, messageIfEmpty) {
```

Comment:
*assume* is also wrong. You call this function to check that that the object is not empty, you don't assume it is.